Ginger/GingerCoreCommon/ALMLib/ALMConfig.cs

```diff
@@ -203,6 +218,20 @@ public ALMIntegrationEnums.eALMType AlmType
             }
         }
 
+        private string mIsTestSuite;
```

Comment:
Please check if mIsTestSuite flag can be used directly from RQMSettings object. If yes, please remove it from here

Reply:
No we need mIsTesSuite Flag on ALmConfig